src/TimeGrid.jsx

```diff
@@ -77,9 +80,23 @@ export default class TimeGrid extends Component {
       this.measureGutter()
     }
 
+    if (this._scrollRatio) {
+      const { content } = this.refs;
+      content.scrollTop = content.scrollHeight * this._scrollRatio;
+      // Only do this once
+      this._scrollRatio = null;
+    }
+
     //this.checkOverflow()
   }
 
+  componentWillReceiveProps(nextProps) {
+    // When paginating, reset scroll
+    if (+nextProps.start !== +this.props.start) {
```

Comment:
I'd only set the scroll if the start time changed _or_ scrollTime changed. 

as an aside I'd also use `dates.eq` using the 'day' unit for the start comparison as well so we don't switch scroll if milliseconds change for instance